packages/aws-serverless/src/utils.ts

```diff
@@ -12,3 +36,47 @@ export function markEventUnhandled(scope: Scope): Scope {
 
   return scope;
 }
+
+/**
+ * Extracts sentry trace data from the handler `context` if available and falls
+ * back to the `event`.
```

Comment:
Maybe we can also add a short note here when this would be on context and when on event?

Reply:
I added an explanation, but tbh this is pretty wide open. I think different AWS services make use of different event/context usage. Hope this adds a bit more info?